bundles/org.openhab.binding.ventaair/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:ventaair:humidifier">
+		<parameter name="ipAddress" type="text" required="true">
+			<label>IP Address</label>
+			<context>network-address</context>
+			<description>IP Address or hostname of the device</description>
+		</parameter>
+		<parameter name="macAddress" type="text" required="true">
+			<label>MAC Address</label>
+			<description>MAC Address of the device</description>
+		</parameter>
+		<parameter name="deviceType" type="integer" required="true">
+			<label>Device Type</label>
+			<description>Type of the device as integer</description>
+		</parameter>
+		<parameter name="pollingTime" type="integer" required="false">
+			<label>Polling interval (s)</label>
```

Comment:
```suggestion
		<parameter name="pollingTime" type="integer" required="true" min="1" max="6000" unit="s">
			<label>Polling Interval</label>
```
Since your using a method called validateConfig() to ensure that the value is not null and above 0, you can do it here with these changes.